wtransport/src/tls.rs

```diff
@@ -685,15 +686,53 @@ pub mod client {
     impl ServerCertVerifier for NoServerVerification {
         fn verify_server_cert(
             &self,
-            _end_entity: &rustls::Certificate,
-            _intermediates: &[rustls::Certificate],
-            _server_name: &rustls::ServerName,
-            _scts: &mut dyn Iterator<Item = &[u8]>,
+            _end_entity: &rustls_pki_types::CertificateDer,
+            _intermediates: &[rustls_pki_types::CertificateDer],
+            _server_name: &rustls_pki_types::ServerName,
             _ocsp_response: &[u8],
-            _now: std::time::SystemTime,
+            _now: rustls_pki_types::UnixTime,
         ) -> Result<ServerCertVerified, rustls::Error> {
             Ok(ServerCertVerified::assertion())
         }
+
+        fn verify_tls12_signature(
+            &self,
+            message: &[u8],
+            cert: &CertificateDer<'_>,
+            dss: &rustls::DigitallySignedStruct,
+        ) -> Result<rustls::client::danger::HandshakeSignatureValid, rustls::Error> {
+            rustls::crypto::verify_tls12_signature(
+                message,
+                cert,
+                dss,
+                &CryptoProvider::get_default()
+                    .unwrap()
+                    .signature_verification_algorithms,
```

Comment:
My thinking is that we wont verify certificates until a `ClientConfig` or `ServerConfig` is built, which call `get_default_or_install_from_crate_features`.